ht/cogs/roll.py

```diff
@@ -65,7 +65,7 @@ async def symbolism(self, ctx, user: converters.Armiger = None):
 			
 			if not values:
 				raise utils.CustomCommandError(
-					"Armiger doesn't have symbolisn roll-of-arms.com",
+					"Armiger doesn't have symbolism roll-of-arms.com",
```

Comment:
```suggestion
					"Armiger doesn't have symbolism on roll-of-arms.com",
```
Might as well fix my grammar mistake as well.